src/core/ledger/grain.js

```diff
@@ -60,6 +60,9 @@ export function mul(a: Grain, b: Grain): Grain {
 export function div(a: Grain, b: Grain): Grain {
   return bigInt(a).divide(bigInt(b)).toString();
 }
+export function pow(a: Grain, b: Grain): Grain {
+  return bigInt(a).pow(bigInt(b)).toString();
+}
```

Comment:
This doesn't seem right. Why would we be raising something to a `Grain` power? As an example, let `one` be one grain. Then `Grain.power(one, one)` is `(10^18) ^ (10^18)`. Also this forbids doing non-integer powers which frankly are the powers I"m most interested in (e.g. sqrt(x) is pow(x, 0.5)).

Also: if adding a new mathematical primitive to the grain module, please make that into its own PR.